src/setup/devnet/index.js

```diff
@@ -381,6 +393,58 @@ export class Devnet {
     })
   }
 
+  async getSnapshotSyncTasks() {
+    return new Listr([
+      {
+        title: 'Download heimdall snapshot',
+        task: async () => {
+          for (let i = 0; i < this.totalNodes; i++) {
+            await execa(
+              'ssh',
+              [
+                '-o',
+                'StrictHostKeyChecking=no',
+                '-o',
+                'UserKnownHostsFile=/dev/null',
+                '-i',
+                '~/cert.pem',
+                  `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                  `mkdir -p ~/.heimdalld/data && sudo curl ${this.config.heimdallSnapshot} | sudo tar zxf - -C ~/.heimdalld/data && sudo chmod 777 -R ~/.heimdalld/data`
```

Comment:
Don't we need to stop `heimdall` prior to this?

Reply:
That's good point! I missed it but I didn't encounter any issues. I'll make a note, though.

Reply:
Yeah please let's add that stop command for heimdall too

Reply:
Done: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)